tools/python/util/convert_onnx_models_to_ort.py

```diff
@@ -250,6 +266,7 @@
 def convert_onnx_models_to_ort():
     args = parse_args()
 
+    output_dir = args.output_dir.resolve() if args.output_dir else None
     optimization_styles = [OptimizationStyle[style_str] for style_str in args.optimization_style]
```

Comment:
Should `resolve` have strict=True, or alternatively post-resolve if the path doesn't exist should we create the directory?